packages/apps-config/src/api/spec/index.ts

```diff
@@ -208,7 +207,6 @@ const spec: Record<string, OverrideBundleDefinition> = {
   'jur-chain': jur,
   kerria: parallel,
   khala,
-  'kilt-spiritnet': kilt,
```

Comment:
The changes as applied look fine - do all the chains above have this spec_name as part of the runtime version? If so, it could just be mapped on spec_name, same as what is done with the chain name now.

Reply:
And the `kilt-spiritnet` chain spec only identifies our live chain.